crates/bevy_ecs/src/system/commands/command_queue.rs

```diff
@@ -34,6 +34,19 @@ pub struct CommandQueue {
     bytes: Vec<MaybeUninit<u8>>,
 }
 
+// CommandQueue needs to implement Debug manually, rather than deriving it, because the derived impl just prints
+// [core::mem::maybe_uninit::MaybeUninit<u8>, core::mem::maybe_uninit::MaybeUninit<u8>, ..] for every byte in the vec,
+// which gets extremely verbose very quickly, while also providing no useful information.
+// It is not possible to soundly print the values of the contained bytes, as some of them may be padding or uninitialized (#4863)
+// So instead, the manual impl just prints the length of vec.
+impl Debug for CommandQueue {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("CommandQueue")
+            .field("len", &self.bytes.len())
```

Comment:
I'd prefer it if the name of this field made it clear that this length is in bytes, but it's not blocking.

Reply:
```suggestion
            .field("len_bytes", &self.bytes.len())
```